graphql_compiler/cost_estimation/cardinality_estimator.py

```diff
@@ -89,41 +171,32 @@ def _estimate_children_per_parent(
 
     Given a parent location of type A and child location of type B, assume all AB edges are
```

Comment:
This docstring can be a bit more general now that we are using vertex_edge_counts. (We do not need to rely on the uniformity assumption).

Reply:
We are still using an assumption for uniformity. Since there's a lot of assumptions involving uniformity, I'll describe the current one. 
We find the number of AB edges (this is where vertex_edge_vertex_counts is used). Then, we assume each A vertex has an equal number of AB edges, and we finally get that for one A vertex, there are approximately `(# of AB edges) / (# of A vertices)` edges.

Reply:
Ok nevermind it is correct as it is.